tests/openvino/test_modeling.py

```diff
@@ -933,15 +933,18 @@ def test_beam_search(self, model_arch):
             do_sample=False,
             eos_token_id=None,
         )
+
         beam_sample_gen_config = GenerationConfig(
             max_new_tokens=10,
             min_new_tokens=10,
             num_beams=4,
             do_sample=True,
             eos_token_id=None,
-            top_k=1,
         )
 
+        if model_arch == "minicpm":
+            beam_sample_gen_config.top_k = 1
```

Comment:
why do we need this for minicpm ?

Reply:
model provides instable results without it